### ISSUE TITLE
None of the code is properly documented

### ISSUE THREAD

**Issue by miller9904** [skylight-software/impulse#10]
I'm terrible at documenting code.  I plan to go back through and fully comment the code for the 0.1 release.

**Comment by elogent** [comment]
So this needs to be done before `release-0.1` is merged to `master`, correct?

Also, I think the documentation we need most at the moment is documentation of the different Git branches. Perhaps that should go in the project wiki?

**Comment by miller9904** [comment]
Yes, it needs to be done before release-0.1 is merged.  And branch documentation would be great, too.

**Comment by elogent** [comment]
As long as we stick to Gitflow, it doesn't need to be much, but a simple note pointing to the Gitflow spec and mentioning any ways our branching model differs from Gitflow would be helpful.

**Comment by miller9904** [comment]
Right.

**Comment by miller9904** [comment]
I created a page.  Feel free to bulk it up if you think it needs more.  https://github.com/skylight-software/impulse/wiki/Gitflow.

**Comment by miller9904** [comment]
OK, I added some more code comments last night with #12.  We still need formal documentation with some kind of documentation generator, but this is a start.

**Comment by miller9904** [comment]
I'm moving this issue to the 0.2 milestone.
